Code/Framework/AzFramework/AzFramework/Terrain/TerrainDataRequestBus.h

```diff
@@ -140,54 +234,32 @@ namespace AzFramework
                 Sampler sampleFilter = Sampler::DEFAULT,
                 bool* terrainExistsPtr = nullptr) const = 0;
 
+            //! Flags for selecting the combination of data to query when querying a list or region.
+            //! The flags determine which subset of data in the SurfacePoint struct will be valid in the FillCallback.
+            enum TerrainDataMask : uint8_t
+            {
+                Heights     = 0b00000001,   //! Query height data
+                Normals     = 0b00000010,   //! Query normal data
+                SurfaceData = 0b00000100,   //! Query surface types and weights
+                All         = 0b11111111    //! Query for every available data channel (heights, normals, surface data)
```

Comment:
To each their own. I include non-existent flags so that I don't have to keep changing the value every time I add a flag.

Side note - I might be adding another flag soon for "TerrainExistsFlags".